_docs/policies/README.md

```diff
@@ -0,0 +1,90 @@
+the creation, amendment, and enforcement of all actions within the
+`status-go` repository. These policies reflect our core values of
+inclusivity, transparency, and consensus-driven decision-making while
+defining enforceable rules that guide `status-go` contributions. Policies
+are not merely guidelines but are to be respected and adhered to, ensuring
+alignment across contributors.
+
+## Upholding Policies Through Consensus
+
+### Collective Agreement
+
+The enforceability of policies stems not from the authority of any
+single individual or group but from the collective agreement and shared
+commitment of all `status-go` contributors. Policies are not imposed
+unilaterally but are the result of transparent discussions and explicit
+recorded approval from key stakeholders. This includes team leads,
+members of the @status-im/status-go-guild GitHub team, and other
+relevant contributors.
+
+### Shared Responsibility
+
+Respecting and adhering to policies is a shared responsibility
+that reflects the values and goals of `status-go` contributors. Approved
+policies are not merely recommendations but agreed-upon standards,
+created through mutual understanding and collaboration, that guide
+how we work together and contribute to the project.
+
+### Mutual Enforcement through Alignment
+
+The power of enforcement does not rest with any one authority; it
+arises from the collective commitment of all contributors to uphold
+policies that have been collaboratively crafted and agreed upon. This
+ensures that policies are respected not out of obligation but because
+they represent the shared vision and trust of the contributors.
+
+### Fostering Alignment
+
+Policies are designed to ensure consistency, fairness, and alignment
+across the teams, creating a framework that supports effective
+collaboration and decision-making. By honouring the principles of
+inclusivity and consensus, we strengthen trust and accountability
+within all contributors.
+
+By grounding our policies in transparency, mutual respect, and collective
+ownership, the `status-go` project ensures they are both enforceable and
+reflective of the shared goals of all contributors.
+
+## Inclusivity
+
+Every `status-go` contributor must have an equal opportunity to
+participate in the creation and review of policies. Contributions
+from all backgrounds and perspectives are encouraged and respected.
+The process must ensure:
+
+- **Open participation**: Any individual or group who interacts
+with or contributes to the project has the right to voice their
+opinions and suggestions.
+- **Accessibility**: Policies must be written in clear, accessible
+language, and the decision-making process should be easy to understand
+for everyone involved.
+
+## Transparency
+
+All discussions, decisions, and processes related to policy creation
+must be fully transparent. This includes:
+
+- **Open documentation**: Discussions leading to policy decisions
```

Comment:
Would be good to have some guidelines on where you intend to hold these discussions and publish the documentation.

From experience, a lot of discussions are happening in private Discord groups or Discord channels.

Reply:
But this clause is specifically about keeping the decisions _public_ 🤔 

In general, it will a PR, just like this one. All comments and resolutions are publicly documented. 
But the README shouldn't limit it to a specific platform. README document describes the intentions, it's not a policy itself.

At the same time `_docs/policies/submitting-policy.md` describes that the policy should be a PR and all feedbacl must be resolved. This covers the intentions described in the README.

@fryorcraken does this make sense?